src/library/scala/collection/Seq.scala

```diff
@@ -540,19 +540,22 @@ trait SeqOps[+A, +CC[_], +C] extends Any
     else new PermutationsItr
 
   /** Iterates over combinations.  A _combination_ of length `n` is a subsequence of
-    *  the original sequence, with the elements taken in order.  Thus, `"xy"` and `"yy"`
-    *  are both length-2 combinations of `"xyy"`, but `"yx"` is not.  If there is
+    *  the original sequence, with the elements taken in order of their first occurrence.  Thus, `"yy"` and `"yx"`
+    *  are both length-2 combinations of `"yxy"`, but `"xy"` is not.  If there is
     *  more than one way to generate the same subsequence, only one will be returned.
     *
-    *  For example, `"xyyy"` has three different ways to generate `"xy"` depending on
+    *  For example, `"yxyy"` has three different ways to generate `"yx"` depending on
     *  whether the first, second, or third `"y"` is selected.  However, since all are
     *  identical, only one will be chosen.  Which of the three will be taken is an
     *  implementation detail that is not defined.
     *
     *  $willForceEvaluation
     *
     *  @return   An Iterator which traverses the possible n-element combinations of this $coll.
-    *  @example  `"abbbc".combinations(2) = Iterator(ab, ac, bb, bc)`
+    *  @example {{{
+    *    "abbbc".combinations(2) = Iterator(ab, ac, bb, bc)
+    *    "bab".combinations(2) = Iterator(bb, ba)
```

Comment:
two nitpicks:

1. `"abbbc"` and `"bab"` are `String`s, not `Seq`s. I know using `Seq`s makes the examples a bit less clean, but accurate is still preferable.
2. aside from the fact that `=` does not use the same semantics as the actual language in these examples (in fact, these examples do not compile because of that, among other reasons), the result of `SeqOps#combinations` as printed in the REPL would be `<iterator>`, because `Iterator`s are lazily evaluated.
   ```scala
   scala> Seq(1, 0, 1).combinations(2)
   val res0: Iterator[String] = <iterator>
   ```
   I'm not sure what best way of representing the results is. options include `Seq(1, 0, 1).combinations(2).toList` => `List(Seq(1, 1), Seq(1, 0))`, as well as possibly using literal REPL output? `==` cannot be used in the example because `Iterator`s use reference equality.

the second nitpick applies to `StringOps#combinations` as well.

Reply:
From an accurate point of view, you are right.

I did not change the style of example since there are many other example is written in pseudo code. And I am not sure which style of example is better, so I just follow previous code.

For example, in `ArrayOps#combinations` doc
```
@example  {{{
   Array("a", "b", "b", "b", "c").combinations(2) == Iterator(Array(a, b), Array(a, c), Array(b, b), Array(b, c))
}}}
```

By the way, I cant find any usage of  `ArrayOps#combinations` in source and test, it's strange.